sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_document_async.py

```diff
@@ -155,7 +151,7 @@ async def analyze_document():
                 )
         if kv_pair.value:
             print(
-                    "Value '{}' found within '{}' bounding regions".format(
+                    "\nValue '{}' found within '{}' bounding regions".format(
```

Comment:
sorry I wasn't clear, I meant \n at the end of the string so hopefully the output would be like:

Key: thing
Value: thing

Key: thing2
Value: thing2

But I'm open to whatever looks good printed on the console.